[26기 전지완] add: 26th Westagram

jeonjiwan94 opened the pull request:
>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
(여기에 수정 사항에 대한 간략한 한줄 요약/제목 작성해 주세요.)
- 로그인 버튼 활성화 기능 구현(예시)
		 
## 수정 사항들 자세한 내용
(여기에 수정 사항에 대한 자세한 내용을 작성해 주세요.)
- id '@' 포함, pw 5 글자 이상 입력 시 버튼 활성화(예시)

## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)
- 함수 로직이 너무 긴 것 같습니다. 좀 더 효율적인 방법이 없을지 궁금합니다.(예시) 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [ ] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [ ] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [ ] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ ] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

yojuyoon commented on students/13th/jiyoung/styles/main.css:
```diff
@@ -0,0 +1,403 @@
+    color: #8E8E8E
+}
+
+.e{
+    display: flex;
+    flex-direction: row;
+
+}
+
+.follow1, .follow2, .follow3{
+    padding: 8px 16px 8px 5px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+
+.bold{
+    font-weight: 700;
+}
+
+
+
+.day {
+
+    padding-left: 16px;
+    font-size: 10px;
+
+}
+
+
+
+input{
+    padding-left:10px;
+    text-align: left;
+    width: 558px;
+    border:None;
+    align-items: center;
+
+}
```
전체적으로 indent나 스페이스 사용에 유의해서 가독성에 신경써주시면 좋을 것 같습니다! 🙂

yojuyoon commented on students/13th/jiyoung/js/login.js:
```diff
@@ -0,0 +1,20 @@
+const bttn =document.getElementById('btn3');
+
+function activateBtn(){
+   
+    if(idd.value.includes("@") && pw.value.length>5) {
+
+       bttn.style.backgroundColor="#bbe0fc";
+       
+    } else {
+
+        bttn.style.backgroundColor="grey";
+    }
+
+}
```
삼항연산자를 이용해보는 방법도 있겠네요! 아래 링크를 참고해주세요!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

yojuyoon commented on students/13th/jiyoung/js/main.js:
```diff
@@ -0,0 +1,84 @@
+
+commment.addEventListener('keyup', (e) => {
+    addComment(e)
+});
+
+
+
+let search = document.querySelector("#검색");
+
+let searchbar = document.querySelector(".검색");
+
+
+function extendSearchbar(e) {
+    
+    
+
+    if (search.value.includes("we"))
+
+    {   
+        let span = document.createElement('span');
+        let div = document.createElement('div');
+
+
+        div.appendChild(span);
+
+        div.className = "wecode";
+        span.className = "wecode_bootcamp";
+        span.innerHTML = "Wecode_bootcamp";
+
+
+
+        searchbar.appendChild(div);
+    
+    }
+
+ return;
+}
+
+
+search.addEventListener('keyup', (e) => extendSearchbar(e))
```
로직 잘 구현해주셨네요! 다만 함수안에서 로직 사이사이에 간격은 불필요한 것 같습니다! 스페이스는 함수와 함수 사이에 넣어주시고 함수 내부에 로직 구현시에는 사용을 지양해주세요! :)

yojuyoon reviewed (commented):
안녕하세요 지영님! 리뷰 내용 참고해서 반영해주세요! 🙌

jeonjiwan94 committed:
[26기 전지완]

전체적인 guideline에 맞게 하려고 노력함. 다만, post button 기능구현에서(댓글 입력)에서 post버튼이 생각대로 작동하지 않는다. StackOverflow에서 질문 했을때 많은 사람들이 코드에 문제가 없고 자신들은 잘 된다고 하는데 직접 체크해봤을때 작동x
- 파일위치 확인 o
- developer tool에서 network에 js파일 전달 되는거 확인 o
- js파일 typeo 확안 o
